server/priv/repo/migrations/20211228014915_add_wal_payload_on_errors_in_apply_rls_function.exs

```diff
@@ -0,0 +1,247 @@
+defmodule Realtime.RLS.Repo.Migrations.AddWalPayloadOnErrorsInApplyRlsFunction do
+  use Ecto.Migration
+
+  def change do
+    execute "create or replace function realtime.apply_rls(wal jsonb, max_record_bytes int = 1024 * 1024)
```

Comment:
@olirice can you please take a quick peek at this and make sure it reflects https://github.com/supabase/walrus/pull/34?

Reply:
confirmed: 0 diff excluding whitespace and comments